diff.py

```diff
@@ -1,47 +1,56 @@
 #!/usr/bin/env python3
 import sys
-import re
-import os
-import ast
-import argparse
-import subprocess
-import difflib
-import string
-import itertools
-import threading
-import queue
-import time
-from typing import Any, Dict, List, NamedTuple, Optional, Set, Tuple, Union
-
 
 def fail(msg):
     print(msg, file=sys.stderr)
     sys.exit(1)
 
-
-MISSING_PREREQUISITES = (
-    "Missing prerequisite python module {}. "
-    "Run `python3 -m pip install --user colorama ansiwrap watchdog python-Levenshtein cxxfilt` to install prerequisites (cxxfilt only needed with --source)."
-)
-
-try:
-    from colorama import Fore, Style, Back  # type: ignore
-    import ansiwrap  # type: ignore
-    import watchdog  # type: ignore
-except ModuleNotFoundError as e:
-    fail(MISSING_PREREQUISITES.format(e.name))
-
 # Prefer to use diff_settings.py from the current working directory
 sys.path.insert(0, ".")
 try:
     import diff_settings
 except ModuleNotFoundError:
     fail("Unable to find diff_settings.py in the same directory.")
+sys.path.pop(0)
 
-# ==== CONFIG ====
+# ==== COMMAND-LINE ====
+
+try:
+    import argcomplete
```

Comment:
```suggestion
    import argcomplete  # type: ignore
```
for mypy's sake

Reply:
what is mypy?

Reply:
https://github.com/python/mypy/

Python type-checker; I use it for most projects. (Just started introducing type annotations into diff.py, so there aren't very many yet.)